compute/is_notifications/notifications/intelligence-stream-significant-impact.adoc

```diff
@@ -10,6 +10,12 @@ The items in this list are intended to keep you informed of only the last 7 brea
 |=========================================================
 |Date Announced |Date Taking Effect |Description of Change
 
+
+| 31 October, 2022 | Lagrange, early December 2022 | The Intelligence stream is being updated to include additional and enhanced risk factor data. with the Kepler Update 3 release slated for Nov 6, 2022.
+
+With this update, if you are on v22.06, you must upgrade to Kepler Update 3 (22.06.xxx) before November 20, 2022. 
+If you do not upgrade, the IS will no longer be able to get the latest vulnerability information from the Palo Alto Networks CDN with the release of the upcoming feature release (code named Lagrange) .
+
```

Comment:
Can you please clarify when IS updates will stop working for 22.06 update 2?  Is it on November 20 or when Lagrange is released?

Reply:
This is work in progress, @cfarquhar. We will let you know.

Reply:
Thanks, MV.  I realize this is a WIP.  I'm just looking to get as far ahead of this as possible.  Since Kepler before update 3 will be impacted, is it safe to assume all versions of Joule will also stop receiving IS updates?

Reply:
no, it does not impact Joule. @cfarquhar